### PR TITLE
Fix #105: Add PRESENCE_CHECK/VERIFICATION_PENDING

### LINKED ISSUE #105: Add PRESENCE_CHECK/VERIFICATION_PENDING
Add state `PRESENCE_CHECK/VERIFICATION_PENDING` to the tests.

Test for https://github.com/wultra/enrollment-server/issues/389

### PR DESCRIPTION
**Pull request by banterCZ**
Depends on https://github.com/wultra/enrollment-server/pull/390

### CODE REVIEW

File: powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java [hunk 1]
```diff
@@ -161,14 +161,15 @@ void setUp() throws IOException {
     }
 
     @Test
-    void testSuccessfulIdentityVerification() throws Exception {
+    void    testSuccessfulIdentityVerification() throws Exception {
```
Comment (romanstrobl) [inline powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java:164]: Formatting issue.

File: powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java [hunk 2]
```diff
@@ -1069,6 +1074,23 @@ private void initPresenceCheck(String processId) throws Exception {
         assertEquals(200, stepLogger.getResponse().getStatusCode());
     }
 
+    private void submitPresenceCheck(final String processId) throws Exception {
+        if (config.isSkipPresenceCheck()) {
+            return;
+        }
+        // TODO (racansky, 2022-09-26) change to PresenceCheckSubmitRequest
+        final PresenceCheckInitRequest presenceCheckRequest = new PresenceCheckInitRequest();
```
Comment (romanstrobl) [inline powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java:1082]: Can we fix this TODO?

**Review by romanstrobl — APPROVED** [review]
I have 2 minor nit-picks, approve otherwise.

File: powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java (hunk 2 above)
Comment (romanstrobl) [reply powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java:1082]: Yes, but you can prepare it already, I think.

Comment (banterCZ) [reply powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java:1082]: Ok, let's wait with merge of enrollment-server after testing app with iProov and then we may fix it.

**Pull request merged**